src/certificates/data/api.js

```diff
@@ -0,0 +1,17 @@
+/* eslint-disable import/prefer-default-export */
+import { camelCaseObject, getConfig } from '@edx/frontend-platform';
+import { getAuthenticatedHttpClient } from '@edx/frontend-platform/auth';
+
+const getApiBaseUrl = () => getConfig().STUDIO_BASE_URL;
+export const getCertificatesApiUrl = (courseId) => `${getApiBaseUrl()}/api/contentstore/v1/certificates/${courseId}`;
```

Comment:
[optional] Please add some space between these constants. This way we will separate the internal constants and the constants for the main functions

Reply:
We are now doing this on the unit course page. Just suggested for your consideration